Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-SB-3-A.yml

```diff
@@ -27,11 +27,13 @@ shortDescription:
 
 #A multi-paragraph description of the activity
 longDescription: 
-  Define static application security testing (SAST) checks suitable to be carried out during the build process, as well as minimum criteria for passing the build - these might differ according to the risk profiles of various applications. Include the respective security checks in the build and enforce breaking the build process in case the predefined criteria is not met. Trigger warnings for issues below the threshold and log these to a centralized system to track them and take actions. If sensible, implement a mechanism to bypass this behaviour if a vulnerability has been accepted or mitigated. However, ensure these cases are explicitely approved first and log their occurence together with a rationale.
+  Define security checks suitable to be carried out during the build process, as well as minimum criteria for passing the build - these might differ according to the risk profiles of various applications. Include the respective security checks in the build and enforce breaking the build process in case the predefined criteria are not met. Trigger warnings for issues below the threshold and log these to a centralized system to track them and take actions. If sensible, implement an exception mechanism to bypass this behaviour if the risk of a particular vulnerability has been accepted or mitigated. However, ensure these cases are explicitely approved first and log their occurence together with a rationale.
```

Comment:
Similarly as before, do you see some other prominent checks?

Reply:
Could be dependency check for instance. I would not fix the type of check here, as this is the point of some of the other practices in the model...